app/src/main/java/org/dharmaseed/android/TalkRepository.java

```diff
@@ -185,6 +187,36 @@ public Cursor getTalksByCenter(
                 DBManager.C.Center.TABLE_NAME + "." + DBManager.C.Center.ID + "=" + venueId);
     }
 
+    /** Check if we have talks downloaded in old/inaccessible locations, and mark them as not downloaded if so
```

Comment:
can you put this comment on the next line?

Reply:
Done!